examples/helloapache/Nulecule.yaml

```diff
@@ -0,0 +1,80 @@
+# Using helloapache as an example
+---
+metadata:
+  name: helloworld 
+  id: helloworld
+  appversion: 0.0.1
+  spec: 1.0.0
+  description: Hello world!
+
+provider:
```

Comment:
I find the use of the key `provider` multiple times quite unfortunate. Wouldn't it make sense to rename this to something like `provider_config`?